paws_collector.js

```diff
@@ -440,6 +440,75 @@ class PawsCollector extends AlAwsCollector {
         this.reportDDMetric('api_throttling', 1)
         return cloudwatch.putMetricData(params, callback);
     };
+
+    /**
+     * Report the client errors and show case on DDMetrics and cloudwatch
+     * @param callback 
+     * @param error 
+     */
+    reportClientError(error, callback) {
+        var cloudwatch = new AWS.CloudWatch({ apiVersion: '2010-08-01' });
+        const params = {
+            MetricData: [
+                {
+                    MetricName: "PawsClientError",
+                    Dimensions: [
+                        {
+                            Name: 'CollectorType',
+                            Value: this._pawsCollectorType
+                        },
+                        {
+                            Name: 'FunctionName',
+                            Value: process.env.AWS_LAMBDA_FUNCTION_NAME
+                        }
+                    ],
+                    Timestamp: new Date(),
+                    Unit: 'Count',
+                    Value: 1
+                }
+            ],
+            Namespace: 'PawsCollectors'
+        };
+        // Tags can be up to 200 characters long
+        let errorMesage = error.errorMessage? `error_message:${error.errorMessage}`:`error_message:not_define`;
+        if(errorMesage.length > 200){
+             errorMesage = errorMesage.slice(0,199);
+        }
```

Comment:
Same here. we don't want to be calling length and slice on something that is not a string. It'll throw an error.

Reply:
Previously I am sending message in string format from each collector , but now I have remove message parameter from tags.